src/components/layouts/Settings.tsx

```diff
@@ -39,102 +39,107 @@ interface AdvanceSetting {
 
 export function Settings(props: Props) {
   const [t] = useTranslation();
+
   const location = useLocation();
 
+  const doesPathNameIncludes = (route: string) => {
+    return location.pathname.includes(route);
+  };
+
   const basic = [
     {
       name: t('company_details'),
       href: '/settings/company_details',
-      current: location.pathname === '/settings/company_details',
+      current: doesPathNameIncludes('/settings/company_details'),
```

Comment:
I think this would work way better if it was checking for start of string.

```ts
location.pathname.startsWith('/settings/company_details')
```

Reply:
changed